### DISCUSSION TITLE
useFieldArray and useController

### DISCUSSION THREAD

**Discussion by SergeyWebDeveloper** [react-hook-form/react-hook-form#5192] [Q&A] [answered]
Hello, @bluebill1049 . Help me please. 

**Describe the bug.**
Instead of empty values, we get the old values.

**To Reproduce**
1. Go to https://codesandbox.io/s/react-hook-form-usefieldarray-forked-lfe8z?file=/src/index.js
2. Remember the data in the form.
3. Click `Delete` button.
4. Click `Append` button.

Am I doing something wrong or is it a bug?

https://user-images.githubusercontent.com/12486958/118268221-cb760b00-b4c5-11eb-8852-481acdda7cae.mp4

**Comment by bluebill1049** [comment, marked as answer]
> defaultValue must be set for all inputs. Supplied defaultValues in the useForm hook will prepare the fields object with default value.

you are missing defaultValue

**Reply by SergeyWebDeveloper** [reply]
I realized what I was doing wrong, thank you!)
